### PR TITLE
Merge 7.2 into master

### PR DESCRIPTION
**Pull request by vasanthrajams**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
- [ ] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process]( https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review).

### CODE REVIEW

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json [hunk 1]
```diff
@@ -193,6 +208,64 @@
...
+        "responses": {
+          "202": {
+            "description": "Started selective key restore operation from the previously stored backup",
+            "headers": {
+              "Retry-After": {
+                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                "type": "integer"
+              },
+              "Azure-AsyncOperation": {
+                "description": "The URI to poll for completion status.",
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json:248]: Should this not be marked as an LRO as well?

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json [hunk 2]
```diff
@@ -808,6 +808,65 @@
...
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the key to get."
+          },
+          {
+            "name": "key-version",
+            "in": "path",
+            "required": true,
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json:830]: I don't believe this is actually required. It's not for other artifact operations.

**Review by heaths — APPROVED** [review]

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json (hunk 1 above)
Comment (vasanthrajams) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json:248]: Selective key restore actually restores multiple versions hence it takes a very long time hence marked as long running operation.

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json (hunk 2 above)
Comment (vasanthrajams) [reply specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json:830]: This is checked in code from 7.2 I'll pass this comment info to the original authors.

**Review by tjprescott — APPROVED** [review]

**Review by heaths — APPROVED** [review]

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json [hunk 3]
```diff
@@ -808,6 +808,63 @@
...
+        "parameters": [
+          {
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the key to get."
+          },
+          {
+            "name": "key-version",
```
Comment (raych1) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json:828]: path parameter should be required.

**Pull request merged**